src/templates/app/shard.yml.ecr

```diff
@@ -6,7 +6,7 @@ authors:
 
 crystal: <%= Crystal::VERSION %>
 
-license: MIT
+license: UNLICENSED
```

Comment:
Rather than UNLICENSED wouldn't it be better off just to delete the field?

Reply:
npm requires a license field and they recommend using "UNLICENSED".  I think for consistency sake we should keep this in.